apps/purchase/urls.py

```diff
@@ -6,9 +6,12 @@
 router = routers.DefaultRouter()
 router.register(r'purchase_orders', PurchaseOrdersViewSet)
 router.register(r'purchaseorder_items', PurchaseorderItemsViewSet)
```

Comment:
change this purchaseorder_items to purchase_order_items

Reply:
Chnaged to purchase_order_items